.github/workflows/run-qa-engine.yml

```diff
@@ -0,0 +1,30 @@
+name: Run QA Engine
```

Comment:
I could not test this workflow as it needs to be on the main branch to be manually triggered.

Reply:
This might be helpful for you: https://github.com/airbytehq/github-workflow-test-repo-base (see readme, topher directed me toward it)